modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -602,4 +603,114 @@ def make_request(example)
+        Check claim status using the GET veterans/{veteranId}/claims/{id} endpoint.
+
+        **A substantially complete 526EZ claim must include:**
+        * Veteran's name
+        * Sufficient service information for VA to verify the claimed service
+        * At least one claimed disability or medical condition and how it relates to service
+        * Veteran and/or Representative signature
+
+        **Standard and fully developed claims (FDCs)**
+
+        [Fully developed claims (FDCs)](https://www.va.gov/disability/how-to-file-claim/evidence-needed/fully-developed-claims/)
+        are claims certified by the submitter to include all information needed for processing. These claims process faster#{' '}
+        than claims submitted through the standard claim process. If a claim is certified for the FDC, but is missing needed information,#{' '}
+        it will be processed as a standard claim.
+
+        To certify a claim for the FDC process, set the claimProcessType to FDC_PROGRAM.
+      VERBIAGE
+      description get_schema_description
+
+      parameter name: 'veteranId',
+                in: :path,
+                required: true,
+                type: :string,
+                example: '1012667145V762142',
+                description: 'ID of Veteran'
+
+      let(:veteranId) { '1013062086V794840' } # rubocop:disable RSpec/VariableName
+      let(:Authorization) { 'Bearer token' }
+
+      let(:scopes) { %w[system/526.override] }
+
+      parameter name: :disability_comp_request, in: :body,
+                schema: SwaggerSharedComponents::V2.body_examples[:sync_disability_compensation][:schema]
+
+      # WE MIGHT NOT NEED BOTH EXAMPLES THIS IS WIP SO JUST HAD THEM IN HERE
+      # EXPECTING THESE DOCS WOULD BE THE SAME AS THE 526, BUT WE MAY GO TO ONE
+      parameter in: :body, examples: {
+        'Minimum Required Attributes' => {
+          value: JSON.parse(Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                            'disability_compensation', 'valid_526_minimum_synchronous.json').read)
+        },
+        'Maximum Attributes' => {
+          value: JSON.parse(Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                            'disability_compensation', 'form_526_json_api_synchronous.json').read)
+
+        }
+      }
+
+      describe 'Getting a successful response' do
+        response '202', 'Successful response' do
+          let(:claim_date) { (Time.zone.today - 1.day).to_s }
+          let(:anticipated_separation_date) { 2.days.from_now.strftime('%Y-%m-%d') }
+          let(:data) do
+            temp = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                   'disability_compensation', 'form_526_json_api_synchronous.json').read
+            temp = JSON.parse(temp)
+            attributes = temp['data']['attributes']
```

Comment:
can we not use the existing validation schema here in the 202 response section and just merge in the `claimId` into this destructured section? Then it should reflect in the documentation correctly, right? And it wouldn't be added to the `form_526` for the initial submit so you wouldn't need the 
```
          if params[:action] == 'synchronous'
            validate_json_schema('526_SYNCHRONOUS')
          else
            validate_json_schema
          end
````

there could be potential flaws with the above but I'm just perusing this from the initial take.